Automattic/jetpack
# ESLint: Simplify via configuration cascading

### Changes proposed in this Pull Request:
Simplify ESLint configuration by cascading our configuration files. This significantly simplifies the `yarn lint` command.

See https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?

No, this does not add/remove features from Jetpack.

#### Testing instructions:

Run `yarn lint` and ensure that it outputs the same results as the current `master` branch.

#### Proposed changelog entry for your changes:


No changelog necessary.
